### DISCUSSION TITLE
Setting up recurring events?

### DISCUSSION THREAD

**Discussion by proffalken** [alfio-event/alf.io#1342] [Q&A]
We have an Open Night on the first Tuesday of every month, and we want to use Alf.io to track the attendance.

The events are identical apart from the date they occur on.

Is there a way to have an event become recurring, or do we need to set it up every time with a new URL such as `/open-night-april-2024` and then `open-night-may-2024` etc?

**Comment by syjer** [comment]
Hi @proffalken , currently there is no possibility to make it recurrent, you will need to create it for each occurrence.

To simplify your life you can use a cron job + a script that call the rest api (see https://github.com/alfio-event/alf.io/wiki/REST-API-v1#create-event ).

Or if you don't mind some small manual intervention, you can always copy a previous event and update the dates.

**Reply by proffalken** [reply]
Ok, thanks - is this something already on a feature list somewhere, or something that would be considered as a future feature?
